src/js/components/componentDiagram/graph/index.js

```diff
@@ -0,0 +1,308 @@
+import dagre from 'dagre';
+import deepmerge from 'deepmerge';
+
+import Geometry from '../../../helpers/geometry.js';
+
+import { createSVGElement, findTraversableNodesAndEdges, prepareNode } from './util.js';
+import ClusterGroup from './groups/clusterGroup.js';
+import NodeGroup from './groups/nodeGroup.js';
+import LabelGroup from './groups/labelGroup.js';
+import EdgeGroup from './groups/edgeGroup.js';
+
+const NODE_PADDING_HORIZONTAL = 15;
+const NODE_PADDING_VERTICAL = 10;
+
+const DEFAULT_OPTIONS = {
+  animation: {
+    enable: true,
+    duration: 300,
```

Comment:
Somewhere between 500-1000ms feels better to me, but this may better be resolved if we were able to override this.

Reply:
Indeed, I made 300ms as default animation speed. But you can set your own duration value like in [`096b5a9`](https://github.com/applandinc/diagrams-js/pull/57/commits/096b5a9882ca7271b069e3f0e802bee595f83245)